cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -115,3 +115,608 @@ def test_validate_not_required_dialog_element():
         1692736
     """
     pass
+
+
+@pytest.mark.tier(2)
+def test_dynamically_popullated_text_area__fields_pass_null_to_ansible_tower_templates():
+    """
+
+    Bugzilla:
+        1696474
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/2h
+        testSteps:
+            1. Add dynamic dialog with "text area" field
+            2. Add service catalog with above created dialog
+            3. Order the service
+        expectedResults:
+            1.
+            2.
+            3. Populated "text area" fields should pass correct value  to ansible tower templates
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_dialog_editor_allows_you_to_modify_field_and_exit_without_saving_dialog():
```

Comment:
Possible to shorten this test case name? How about just `test_dialog_editor`? I don't think the test name needs to specify everything that the test_case does, that's what the testSteps are for.